extensions/omni_schema/migrate/2_aux_tools.sql

```diff
@@ -4,13 +4,14 @@ create table auxiliary_tools
     filename_stem       text,
     filename_extension  text,
     processor           name not null,
-    processor_extension name
+    processor_extension name,
+    priority            integer not null default 0
 );
 
 insert
 into
-    auxiliary_tools (filename_stem, filename_extension, processor, processor_extension)
+    auxiliary_tools (filename_stem, filename_extension, processor, processor_extension, priority)
 values
-    ('requirements', 'txt', 'omni_python.install_requirements', 'omni_python');
+    ('requirements', 'txt', 'omni_python.install_requirements', 'omni_python', 1);
```

Comment:
Question: Do we need order when we load multiple Python files that have some dependency between them? If yes, this approach doesn't handle that (unless I'm missing something!).
Appreciate your thoughts!

Reply:
This approach doesn't handle this case at all. I just wanted to establish some predictable and sane order.

The dependency mechanism would likely involve custom importers and is more on omni_python side than omni_schema:

https://docs.python.org/3/library/importlib.html